File: src/main/java/org/project/model/response/StaffEducationResponse.java
package org.project.model.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class StaffEducationResponse {
    private Long id;
    private Integer year;
    private String degree;
    private String institute;
    private String result;
}

File: src/main/resources/templates/frontend/assets/js/model/response/StaffEducationResponse.js
export class StaffEducationResponse {
    constructor(id, year, degree, institute, result) {
        this.id = id;
        this.year = year;
        this.degree = degree;
        this.institute = institute;
        this.result = result;
    }

    static fromJson(json) {
        return new StaffEducationResponse(
            json.id,
            json.year,
            json.degree,
            json.institute,
            json.result
        );
    }

    static fromJsonArray(jsonArray) {
        return jsonArray.map(json => StaffEducationResponse.fromJson(json));
    }
}
